src/lib/components/icon/Icon.component.tsx

```diff
@@ -79,36 +90,51 @@ export const iconTable = {
   'Log-out': 'fas faSignOutAlt',
   Hourglass: 'far faHourglass',
   Pause: 'fas faPause',
+  'Pause-circle': 'far faPauseCircle',
+  'Play-circle': 'far faPlayCircle',
   Upgrade: 'fas faLevelUpAlt',
   Expansion: 'fas faExpandAlt',
   Rebalance: 'fas faBalanceScale',
   Maintenance: 'fas faHardHat',
+  Role: 'fas faHatCowboy',
   'Change-erasure': 'fas faExchangeAlt',
   'Circle-health': 'fas faCircle',
   'Circle-empty': 'far faCircle',
+  'Star-filled': 'fas faStar',
```

Comment:
Same, I don't remember Star icon.

Reply:
It really, really doesn't ring a bell... I think it's only on Core-ui, but not in ARTESCA. I would remove it if it's the case.